### PR TITLE
Edit user bio and displayname

### PR DESCRIPTION
**Pull request by JakeSteinebronn**
# Changes
Implement functionality in frontend and backend for:
- Editing user bio
- Editing user displayname
- Display error messages if bio and/or displayname are invalid
- Reorder the modal so that the image input is on the bottom, because otherwise you can't see the text input because the mobile keyboard pops up and covers it :(
# Issue ticket number and link
#112

### CODE REVIEW

**Review by tylerhm — CHANGES_REQUESTED** [review]
I think the single debounce session is broken in certain cases.

1. User types some info in bio
2. User types some info in displayName less than 500ms after stopping in bio
3. After 500ms, only displayName updates

**Comment by abrugal** [comment]
> I think the single debounce session is broken in certain cases.
> 
> 1. User types some info in bio
> 2. User types some info in displayName less than 500ms after stopping in bio
> 3. After 500ms, only displayName updates

I couldn't reproduce this, it seems to work fine for me

**Comment by tylerhm** [comment]
I haven't had a chance to test locally yet, but the code seems wrong.

Whatever callback is sent to the debounce session overwrites the previous one if it hasn't run yet. If they are responsible for mutating different parts of the state then this is definitely a bug here.

**Comment by tylerhm** [comment]
https://user-images.githubusercontent.com/36250052/216234048-97fc1b7a-279a-42d5-97e8-0046605e208a.mov

I've attached a repro for the described bug.

**Comment by JakeSteinebronn** [comment]
how do I mark the fact that I made the requested changes and I need to be reviewed again?

**Comment by tylerhm** [comment]
I'm not sure if there is a way. Adding commits and a comment is enough though.

File: utils/queries.tsx [hunk 1]
```diff
@@ -29,7 +29,7 @@ export interface FetchedUser {
 }
 export const buildUserFetcher = (user: User) => {
   return async () => {
-    await user.getData();
+    await user.getData(true);
```
Comment (tylerhm) [inline utils/queries.tsx:32]: Oh true, should we do this for all fetchers?

**Review by tylerhm — APPROVED** [review]
saweeet

File: utils/queries.tsx (hunk 1 above)
Comment (JakeSteinebronn) [reply utils/queries.tsx:32]: Probably yes, since now it's pretty freaking hard to abuse this and burn the database to the ground :)

**Pull request merged**